do_snowDA.sh

```diff
@@ -16,22 +16,24 @@
 
 # user directories
 
-WORKDIR=${WORKDIR:-"/scratch2/BMC/gsienkf/Clara.Draper/workdir/"}
-SCRIPTDIR=${DADIR:-"/scratch2/BMC/gsienkf/Clara.Draper/gerrit-hera/AZworkflow/landDA_workflow/"}
-OBSDIR=/scratch2/BMC/gsienkf/Clara.Draper/data_AZ/
-OUTDIR=${OUTDIR:-${SCRIPTDIR}/../output/} 
+WORKDIR=${WORKDIR:-"/scratch1/NCEPDEV/global/Jiarui.Dong/JEDI/workflow/experiment/workdir"}
```

Comment:
As I said in the instructions, before submitting the PR you need to change all optional variables back to the original values.

Reply:
Changed back